Source/AwsCommonRuntimeKit/mqtt/Mqtt5Packets.swift

```diff
@@ -237,8 +241,10 @@ public class PublishResult: @unchecked Sendable {
     }
 }
 
+// We can't mutate this class after initialization. Swift can not verify the sendability due to the class is non-final,
+// so mark it unchecked Sendable
 /// "Data model of an `MQTT5 PUBACK <https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901121>`_ packet
-public class PubackPacket {
+public class PubackPacket: @unchecked Sendable {
```

Comment:
Why is this a class? Can we just make it a struct with Sendable instead of Unchecked Sendable? We should verify all the classes.

Reply:
Normally, we have classes because C Pointers require a deinit function which structs can't have it.

Reply:
It's a little wired to say so, but the fundamental issue was I didnt found a clean way to handle an array of objects without storing raw data inside the class, as a result a `deinit` is required to properly deallocate that data. 
For this reason, any CStruct would potentially need a `deinit` if we would like to convert an Array of the object into c. I'm currently trying to find a good way to handle it the array conversion...

Reply:
> I didnt found a clean way to handle an array of objects without storing raw data inside the c
This class doesn't have any raw data